tools/wavetool.py

```diff
@@ -242,11 +288,11 @@ def parse_cmdline():
     help='sample bits of generated wave')
     parser.add_argument('-o', '--output', type=str, help='path to store generated files', default='.')
     # wave comparison arguments
-    parser.add_argument('-a', '--analyze', type=str, choices=['smart_amp', 'wov'], help='analyze recorded wave to give case verdict')
+    parser.add_argument('-a', '--analyze', type=str, choices=['smart_amp', 'wov', 'thdn', 'alsabat'], help='analyze recorded wave to give case verdict')
```

Comment:
what is alsabat mode here? Does it means we have some predefined parameters here

Reply:
No, The alsabat mode here means that we will skip the ramp, and start analysis from stable state.

Reply:
OK, so this is only some configuration for test right? It is not some algorithm like `thdn`, right? So can we make the configurations more straightforward in some static format data.